docs/tutorials/advanced/rule_based_profilers/how-to-create-a-new-expectation-suite-using-rule-based-profilers.md

```diff
@@ -0,0 +1,130 @@
+
+#### 6. Adding a Rule for Columns
+Let's add one more rule to our Rule-Based Profiler config. This Rule will use the DomainBuilder to populate a list of all of the numeric columns in one Batch of taxi data (in this case, the most recent Batch). It will then use our NumericMetricRangeMultiBatchParameterBuilder looking at the five Batches prior to our most recent Batch to get probable ranges for the min and max values for each of those columns. Finally, it will use those ranges to add two ExpectationConfigurations for each of those columns: `expect_column_min_to_be_between` and `expect_column_max_to_be_between`. This rule will go directly below our previous rule.
+
+As before, we will first add the name of our rule, and then specify the DomainBuilder.
+```yaml file=../../../../tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py#L43-L53
+```
+In this case, our DomainBuilder configuration is a bit more complex. First, we are using a SimpleSemanticTypeColumnDomainBuilder. This will take a table, and return a list of all columns that match the `semantic_type` specified - `numeric` in our case.
+
+Then, we need to specify a BatchRequest that returns exactly one Batch of data (this is our `data_connector_query` with `index` equal to `-1`). This tells us which Batch to use to get the columns from which we will select our numeric columns. Though we might hope that all our Batches of data have the same columns, in actuality, there might be differences between the Batches, and so we explicitly specify the Batch we want to use here.
+
+After this, we specify our ParameterBuilders. This is very similar to the specification in our previous rule, except we will be specifying two NumericMetricRangeMultiBatchParameterBuilders to get a probable range for the `min_value` and `max_value` of each of our numeric columns. Thus one ParameterBuilder will take the `column.min` `metric_name`, and the other will take the `column.max` `metric_name`.
+```yaml file=../../../../tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py#L54-L78
+```
+
+Finally, we'll put together our `Domains` and `Parameters` in our `ExpectationConfigurationBuilders`
+```yaml file=../../../../tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py#L79-L97
+```
+
+Putting together our entire config, with both of our Rules, we get 
+```yaml file=../../../../tests/integration/docusaurus/rule_based_profiler/multi_batch_example.py#L9-L97
+```
+
+And if we re-instantiate our `Profiler` with our config which now has two rules, and then we re-run the `Profiler`, we'll have an updated suite with a table row count expectation for our table, and column min and column max expectations for each of our numeric columns!
+
+#### 🎉Congratulations! You have successfully Profiled multi-batch data using a Rule-based Profiler. Now you can try adding some new rules, or running your Profiler on some other data (remember to change the BatchRequest in your config)!🎉
```

Comment:
Please change this to non-header text.

Reply:
Below the congrats text, please add an Additional Notes section with a link to the script so users may see the entire script on GitHub. This allows them to make comments, PRs directly into our existing feedback processes on GH.

It will look similar to this:
```
## Additional Notes

To view the full scripts used in this page, see them on GitHub:

- [mysql_yaml_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_yaml_example.py)
- [mysql_python_example.py](https://github.com/great-expectations/great_expectations/blob/develop/tests/integration/docusaurus/connecting_to_your_data/database/mysql_python_example.py)
```

(From https://knoxpod.netlify.app/docs/guides/connecting_to_your_data/in_memory/spark#additional-notes)